docker/dotcms-seed/Dockerfile

```diff
@@ -1,20 +1,29 @@
 # OpenJDK distributed under GPLv2+Oracle Classpath Exception license (http://openjdk.java.net/legal/gplv2+ce.html)
 # Alpine Linux packages distributed under various licenses including GPL-3.0+ (https://pkgs.alpinelinux.org/packages)
 # dotCMS core distributed under GPLv3 license (https://github.com/dotCMS/core/blob/master/license.txt)
-FROM adoptopenjdk/openjdk11:alpine as dotcms-seed
+FROM adoptopenjdk/openjdk11:ubuntu as dotcms-seed
 
 LABEL com.dotcms.contact "info@dotcms.com"
 LABEL com.dotcms.vendor "dotCMS LLC"
 LABEL com.dotcms.description "dotCMS Content Management System"
 
 WORKDIR /srv
 
+# Export gradle environment
+
+ENV GRADLE_USER_HOME=/build/src/core/dotCMS/.gradle
+ENV GRADLE_OPTS="-Dfile.encoding=utf-8 -Xmx2g"
+
 # Build env dependencies
-RUN apk update && apk --no-cache add bash grep openssl git sed
+# Installing basic packages 
+RUN apt update && \
+    apt upgrade -y && \
+    apt install -y bash grep openssl git sed
 
 RUN mkdir -p /build/src \
     && echo "Pulling dotCMS src" \
     && cd /build/src && git clone https://github.com/dotCMS/core.git core \
```

Comment:
This is the real benefit of `dotcms-seed`. It will "cache" the `core` repo at `master` in a given point in time.
We have to be careful whenever we do fundamental changes to it to not forget to build it again and reset it to new point in time. I was thinking about of getting rid of it but it turns out it saves a bunch of time caching most of the base code and later on the extending image stage just do a checkoout.